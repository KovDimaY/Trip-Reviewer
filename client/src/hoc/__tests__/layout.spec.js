import React from 'react';
import { create } from 'react-test-renderer';
 
import Layout from './../layout';

jest.mock('./../../components/Header/index', () => 'Header');

const mockComponent = props => {
    return (
        <Layout {...props} />
    );
};
 
describe('<Layout />', () => {
    it('should render component', () => {
        const props = {
            children: <div>test</div>
        };
        const tree = create(mockComponent(props)).toJSON();
        
        expect(tree).toMatchSnapshot();
    });
});