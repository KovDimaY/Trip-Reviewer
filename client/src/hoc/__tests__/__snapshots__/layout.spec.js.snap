// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render component 1`] = `
<div>
  <Header />
  <div
    style={
      Object {
        "minHeight": "95vh",
      }
    }
  >
    <div>
      test
    </div>
  </div>
  <footer
    className="footer-wrapper"
  >
    <div
      className="footer-content"
    >
      <div
        className="social-media-icons"
      >
        <a
          className="footer-github"
          href="https://github.com/KovDimaY/Trip-Reviewer"
          rel="noopener noreferrer"
          target="_blank"
        >
          <i
            className="fab fa-github"
          />
        </a>
        <a
          className="footer-facebook"
          href="https://www.facebook.com/dmytro.kovalenko.1004"
          rel="noopener noreferrer"
          target="_blank"
        >
          <i
            className="fab fa-facebook"
          />
        </a>
        <a
          className="footer-vk"
          href="https://www.vk.com/id11545172"
          rel="noopener noreferrer"
          target="_blank"
        >
          <i
            className="fab fa-vk"
          />
        </a>
        <a
          className="footer-linkedin"
          href="https://www.linkedin.com/in/kovalenkodmytro"
          rel="noopener noreferrer"
          target="_blank"
        >
          <i
            className="fab fa-linkedin"
          />
        </a>
      </div>
      <p
        className="author"
      >
        Copyright Â© Dmytro Kovalenko ;)
      </p>
    </div>
  </footer>
</div>
`;
